client/.eslintrc.js

```diff
@@ -8,6 +8,7 @@ module.exports = {
   env: {
     browser: true,
     jquery: true,
+    es2020: true,
```

Comment:
why

Reply:
regex matchAll

Reply:
Note that I don't think this changes our browser compatibility because the target is still es2018. I imagine matchAll is getting expanded to es2018 compatible code

Reply:
Actually I don't think this is true. I'll make a PR that switched away from using matchAll